### PR TITLE
[Tracker Alignment] All-in-one tool: fix reading of sqlite files

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:
It was noticed that the commit f0d34bf760a35c90f7b2ac4241f31227b4dc799a introduced in PR #24952 actually breaks the possibility to read SQLite files from local afs storage which is a mandatory feature of the tool. This PR reverts it. 
In addition I instrument the unit test of this package to be able to spot issues with this functionality.

#### PR validation:

It relies on the existing unit test.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

This PR is not a backport, but will be backported in all the open release cycle where alignment campaigns are active.

cc:
@vbotta @connorpa @adewit

### CODE REVIEW

**Comment by mmusich** [comment]
type bug-fix

**Comment by christopheralanwest** [comment]
please test

**Comment by pohsun** [comment]
+1

**Comment by silviodonato** [comment]
+1

File: Alignment/OfflineValidation/python/TkAlAllInOneTool/helperFunctions.py [hunk 1]
```diff
@@ -207,36 +206,29 @@ def conddb(*args):
...
-    try:
-        with NamedTemporaryFile(bufsize=0) as sys.stdout:
-            exec(conddbcode, namespace)
-            namespace["main"]()
-            with open(sys.stdout.name) as f:
-                result = f.read()
-    finally:
-        sys.argv[:] = bkpargv
-        sys.stdout = bkpstdout
+    with NamedTemporaryFile(bufsize=0) as sys.stdout:
```
Comment (makortel) [inline Alignment/OfflineValidation/python/TkAlAllInOneTool/helperFunctions.py]: Apparently this change introduced a unit test failure in python3 IB, see https://github.com/cms-sw/cmssw/issues/29337.

**Pull request merged**